workflows/parsl_csa/example_params_files/deepcdr_csa_params.ini

```diff
@@ -0,0 +1,50 @@
+[DEFAULT]
+input_dir = ./csa_data/raw_data
+y_col_name = auc
+hyperparameters_file = ./hyperparameters_default.json
+only_cross_study = False
+epochs = 1000
+
+# Resources
+available_accelerators = ["0","1","2","3","4","5","6","7"]
+; available_accelerators = ["4","5","6","7"]
+; available_accelerators = ["6","7"]
+
+# Model defs
+model_name = deepcdr
+model_scripts_dir = /nfs/ml_lab/projects/improve/data/experiments/apartin/DeepCDR
```

Comment:
Instead of providing full local paths, we can do this:
model_scripts_dir = ../DeepCDR

Reply:
@priyanka9991 Since we assume the Parsl scripts are launched from `IMPROVE/workflows/parsl_csa`, I think we probably need `../../../DeepCDR`. Is that right? These are just examples files. I don't think should enforce anything here. I suggest keeping the full path, especially because Andreas is going to combine his and this version.